config/locales/en-GB.yml

```diff
@@ -212,6 +212,11 @@ en-GB:
         status: Status
         manage: Manage
       form:
+        business_details:
+          abn: "Company Number"
+          abn_placeholder: "eg. 99123456789"
+          acn: "ACN"
```

Comment:
It makes sense for ABN to become Company Number for the UK, but I don't think we have an alternate equivalent for ACN...